Lab#2/index.html

```diff
@@ -0,0 +1,82 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/base.css">
+	<link rel="stylesheet" href="css/640px.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" 
+          integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Awesome Free PSD tempaltes & Resources</title>
+</head>
+<body>
+        <nav class="navigation">
+            <ul class="hidden-panel">
```

Comment:
Better to use (according to BEM): 
<ul class="navigation__panel navigation__panel--hidden">